### ISSUE TITLE
IIS Server Variables security

### ISSUE THREAD

**Issue by dtivel** [MicrosoftDocs/iis-docs#1008]
When I searched the internet for "IIS Server Variables", I found 2 documents:

* https://learn.microsoft.com/en-us/previous-versions/iis/6.0-sdk/ms524602(v=vs.90)?redirectedfrom=MSDN#security
* https://learn.microsoft.com/en-us/iis/web-dev-reference/server-variables

The first link is for a previous version that is not being updated.

However, notably, the [Security section](https://learn.microsoft.com/en-us/previous-versions/iis/6.0-sdk/ms524602(v=vs.90)?redirectedfrom=MSDN#security) in the older document isn't present in the newer document.  The advice seems pretty good and timeless.  Are the security concerns no longer relevant in the current version?  (Why doesn't the newer document have a similar Security section?)

**Comment by lextm** [comment]
I agree adding the security section is a good idea, but nowadays (after 2020) this should become common knowledge of web security.

>  The IIS 6 article should have been written before 2003.
